Makefile

```diff
@@ -147,6 +147,10 @@ add-license-headers: $(GO_ADD_LICENSE)
 check-generate:
 	@hack/check-generate.sh $(REPO_ROOT)
 
+.PHONY: check-plutono-dashboards
+check-plutono-dashboards:
+	@hack/check-plutono-dashboards.sh
```

Comment:
In the original PR, this check is also performed in the `make check` call here:

https://github.com/gardener/gardener/pull/8065/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R172

Why is that change no longer part of this PR?